main.js

```diff
@@ -103,15 +103,24 @@ app.on('ready', () => {
   })
 })
 
+const WIN32 = process.platform === 'win32'
+
 ipcMain.on('CHECK_FOR_UPDATE', function (event) {
   console.log('CHECKING')
```

Comment:
Currently in the beta process, I leave console logs in as my error reporting system shows a breadcrumb trail of logs. This helps me understand how to reproduce certain errors.